packages/ra-core/src/controller/input/ReferenceInputController.spec.tsx

```diff
@@ -1,531 +1,71 @@
 import React from 'react';
```

Comment:
Why is there still a unit test of the component? Shouldn't all of them be migrated to the hook?

Reply:
It's an integration test to ensure all hook are properly integrated